src/scripts/utils/base64Utils.js

```diff
@@ -0,0 +1,12 @@
+export function encode(data){
+  let buff = Buffer.from(data);
```

Comment:
`Buffer` is a Node.js thing, isn't it?

Reply:
Yes, it is. If this causes any incompatibility problem [here](https://base64.guru/developers/javascript/examples/unicode-strings) is another good solution

Reply:
Could you try that? This is all browser based, so Buffer wouldn't work.

Reply:
Done!